src/main/kotlin/no/nav/syfo/melding/kafka/producer/DialogmeldingBestillingMetric.kt

```diff
@@ -28,3 +34,18 @@ val COUNT_KAFKA_CONSUMER_PAMINNELSE_BESTILLING_SENT: Counter =
     Counter.builder(KAFKA_CONSUMER_PAMINNELSE_BESTILLING_SENT)
         .description("Counts the number of paminnelse bestillinger sent to topic - $BEHANDLER_DIALOGMELDING_BESTILLING_TOPIC")
         .register(METRICS_REGISTRY)
+
+val COUNT_KAFKA_CONSUMER_FORESPORSEL_TILLEGGSOPPLYSNING_BESTILLING_SENT: Counter =
```

Comment:
Hvorfor heter de forresten `consumer`? Dettte er jo produsering? 🤔

Reply:
Spørsmålet er om vi burde endre navnene på de gamle, for da mister vi jo historikken på de tellingene 🤔 Men jeg fikser på de nye, hvertfall!

Reply:
Er kanskje ikke så farlig med historikk på disse tellingene, tenker jeg. Vi har jo tallene i databasen. Så gjerne fiks navnene på de gamle også.